File: Problems/C++/Easy/Array/1929. Concatenation of Array.cpp
/*
Runtime: 18 ms, faster than 26.38% of C++ online submissions for Concatenation of Array.
Memory Usage: 12.6 MB, less than 71.84% of C++ online submissions for Concatenation of Array.
*/
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> ans = nums;
        for(int num: nums){
            ans.push_back(num);
        }
        return ans;
    }
};

File: Problems/Python/Easy/Array/1929. Concatenation of Array.py
'''
Runtime: 96 ms, faster than 65.27% of Python3 online submissions for Concatenation of Array.
Memory Usage: 14.1 MB, less than 98.71% of Python3 online submissions for Concatenation of Array.
'''
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans = nums + nums
        return ans
        